distributed-databases/pinot/variables.tf

```diff
@@ -0,0 +1,34 @@
+variable "name" {
+  description = "Name of the VPC and EKS Cluster"
+  type        = string
+  default     = "pinot-on-eks"
+}
+
+variable "region" {
+  description = "region"
+  type        = string
```

Comment:
default value should be set to `us-west-2`

Reply:
Done